### ISSUE TITLE
Revamp Landing Page

### ISSUE THREAD

**Issue by jasonLaster** [devtools-html/debugger.html#806]
I'd like to take a stab at revamping the [old landing page](https://cloud.githubusercontent.com/assets/254562/18764292/b406445a-80de-11e6-8256-b84702bb25e1.png).

There are a couple goals:
* **Help new contributors initial experience**
  * The firefox cli command and about:config step can be confusing
  * The "first time" firefox page walks them through the steps.
* **Help Devs**
  * Remembering how to start browsers can be confusing
  * The browser pages have a launch button so you don't have to think about it

----

![landing page](https://cloud.githubusercontent.com/assets/254562/18764884/54264c8a-80e1-11e6-86ce-ead36e099499.jpg)

**Comment by jlongster** [comment]
Sounds good. One thing to keep in mind is that we want to pull the fake toolbox out into a separate project so the console and other tools can use it when developing locally. This could get too debugger specific (the console won't be able to target other servers yet). I think if we keep the designs simple enough it'll be easier to adapt across projects, and we also don't want to take on a project that is re-creating `about:debugging` (go to `about:debugging` in Firefox). Basically: the simpler the better in my opinion, but some styling and more instructions would be good.

It might be good to pull in bgrins/linclark early on this as we want them to adopt this, and bgrins has been helping prepare other tools for running in a tab.

**Comment by jlongster** [comment]
Also, you don't need to go to `about:config`. Users just need to open the devtools and turn on "Enable browser chrome and addon debugging toolboxes" and "Enable remote debugging" in the Advanced Settings. That's a lot easier than `about:config`.

**Comment by jlongster** [comment]
> The browser pages have a launch button so you don't have to think about it

I'm a little hesitant to make it this complex. We'd basically be re-creating `about:debugging` at this point which is a pretty complex project. We're 2 people working on a full debugger and I'd rather spend our time on more important features.

**Comment by jasonLaster** [comment]
Thanks for the feedback:

> separate project

* I believe the can be pulled into an `RDP Client` project
* I'm hesitant to call it a toolbox as that implies the tool navigation aspect as well as the client connection. *overloaded words sigh*

> about:debugging

* thanks for mentioning, I had not considered the overlap. 
  * This will be important to consider if we consider connecting to service workers, i'm not sure if there's an RDP method that exposes them. 
  * Addons is another story, and my assumption is its not relevant for the standalone case.
* the main difference is that the focus here is on multiple targets (chrome, node) and perhaps launching a browser or invoking a script.

> bgrins/linclark

* absolutely. @bgrins, @linclark is there specific console feedback or general tool feedback? I'd be happy to talk when I get back from PTO (2 mondays from now).
* I assume that the frontend can be somewhat configurable:
  * exclude chromium panels
  * add additional panels
  * change Title to "Console"

> about:config

* I'm torn here as it's also nice to turn off the prompt message
* Also, there are more steps for using the UI (open devtools, open setting, find advanced mode, ...). I don't know, I tried to avoid this with selenium + creating a new profile, but neither marionette nor selenium at the moment is robust. (we probably could fix this though). Also, selenium has a nice feature for finding firefox in the directories)

> launch button

* I assume it could be as easy as an exec check and perhaps a ps to see if it's running... Definitely not something that I plan on putting too much time on, but you're right that these things can become more complex.

* this is something that probably won't be included in V1, which is just a re-skin.


----

Thanks for the feedback. It's good to think through how we want the developer / contributor UX to work as it will have a significant impact. Lets plan a time to talk with Helen and Bryan as well so that we can talk through some of the options. 

I'll hopefully be sharing a re-skin in a bit. It shouldn't change the current UX, so it shouldn't be blocked by these discussions.

**Comment by wldcordeiro** [comment]
> I'm torn here as it's also nice to turn off the prompt message

I always thought this should be surfaced in Firefox proper, the prompt should have "don't ask me again checkbox"

**Comment by jlongster** [comment]
Yead, there's a bug to turn it off if when using `localhost`, which solves the issue. If you're talking about automatically starting with `npm run firefox` directly accessing the prefs makes sense, but for people who just want to make an existing running version of Firefox work opening the tools and clicking 2 checkboxes is pretty easy.

(didn't feel the need to respond to all your points, they are all good)
